frontend/packages/dev-console/integration-tests/tests/serverless.scenario.ts

```diff
@@ -0,0 +1,110 @@
+    await switchPerspective(Perspective.Administrator);
+    expect(sideHeader.getText()).toContain('Administrator');
+  });
+
+  afterEach(() => {
+    checkLogs();
+    checkErrors();
+  });
+
+  it('Increase the Machine sets count', async() => {
+    await browser.wait(until.presenceOf(sidenavView.navSectionFor('Compute')));
+    await sidenavView.clickNavLink(['Compute', 'Machine Sets']);
+    await crudView.isLoaded();
+    
+    await browser.wait(until.elementToBeClickable(firstKebabMenu));
+    await firstKebabMenu.click();
+    browser.wait(until.elementToBeClickable(crudView.actionForLabel("Edit Count")));
+    crudView.actionForLabel("Edit Count").click()
+    
+    await browser.wait(until.elementToBeClickable(machineSetsInput));
+    await machineSetsInput.sendKeys(protractor.Key.chord(protractor.Key.CONTROL, "a"));
+    await machineSetsInput.sendKeys("6");
+    await browser.wait(until.elementToBeClickable(submitCount));
+    await submitCount.click();
+    expect(true).toBeTruthy(); 
+  });
+
+  it('displays subscription creation form for selected Operator', async () => {
+    await crudView.isLoaded();
+    await browser.wait(until.presenceOf(sidenavView.navSectionFor('Operators')));
+    await sidenavView.clickNavLink(['Operators', 'OperatorHub']);
+    await crudView.isLoaded();
+    await catalogView.categoryTabsPresent();
+    await catalogView.categoryTabs.get(0).click();
+    await catalogPageView.catalogTileFor('OpenShift Serverless Operator').click();
+    await browser.wait(until.visibilityOf(installServerlessOperator));
+    await installServerlessOperator.element(by.linkText('Install')).click();
+    await operatorHubView.createSubscriptionFormLoaded();
+
+    expect(operatorHubView.createSubscriptionFormName.getText()).toEqual(
+      'OpenShift Serverless Operator',
+    );
+    await browser.wait(until.visibilityOf(operatorHubView.createSubscriptionFormInstallMode));
+    await operatorHubView.allNamespacesInstallMode.click();
+    
+    expect(operatorHubView.createSubscriptionError.isPresent()).toBe(false);
+    expect(operatorHubView.createSubscriptionFormBtn.getAttribute('disabled')).toEqual(null);
+    await operatorHubView.createSubscriptionFormBtn.click();
+    await crudView.isLoaded();
+    await browser.get(`${appHost}/operatorhub/ns/${testName}`);
+    await crudView.isLoaded();
+    await catalogPageView.clickFilterCheckbox('installState-installed');
+    expect(catalogPageView.catalogTileFor('OpenShift Serverless Operator').isDisplayed()).toBe(true);
+  });
+
+  it(`displays Operator in "Cluster Service Versions" view for "${testName}" namespace`, async () => {
+    await retry(() => catalogPageView.catalogTileFor('OpenShift Serverless Operator').click());
+    await browser.wait(until.presenceOf(installServerlessOperator), 1000).then(() => browser.sleep(500));
+    await operatorHubView.viewInstalledOperator();
+    await crudView.isLoaded();
+    browser.sleep(10000);
```

Comment:
arbitrary sleeps are really bad as they add extra time to tests unnecessarily.
You should be waiting on something specific instead.

Reply:
@christianvogt 
While installing OpenShift Serverless Operator, it installs some other operator as well and the it takes time to load the OpenShift Serverless Operator on the Installed Operator page. The waiting time is also dynamic. I tried to use the browser.wait but it didn't work for me.

Reply:
You need to know what you're waiting for.
This is how flakes happen and start to affect all other tests.

Reply:
I will try to use something which should not affect the other tests.